proxy/logging/LogBuffer.cc

```diff
@@ -191,6 +191,32 @@ LogBuffer::~LogBuffer()
   LogBuffer::checkout_write
   -------------------------------------------------------------------------*/
 
+LogBuffer::LB_ResultCode
+LogBuffer::add_entry(size_t *write_offset, size_t write_size)
```

Comment:
Maybe call this fast_write()?  Add a comment that it's not thread safe and/or only for fast mode?